man/org.freedesktop.systemd1.xml

```diff
@@ -268,6 +270,10 @@ node /org/freedesktop/systemd1 {
                  o job,
                  s unit,
                  s result);
+      JobRemoved2(u id,
```

Comment:
so if we fiddle with this, I think it would be good to take this one step further and add one more thing at the same time: the invocation ID associated with the unit at the moment the job completed/failed, u.e. the unit's invocation_id field. I think that would be a major useful addition, as it means you can issue a start job, get a job id back, then once the job is complete you get the invocation id and that you can then use to watch the service's iteration you triggered.

the addition would be trivial to add. just copy the id into the signal.

Reply:
makes sense, added and renamed